src/module.ts

```diff
@@ -446,7 +446,7 @@ declare module 'h3' {
       from: '#internal/nuxt-simple-robots',
       imports: [
         'getPathRobotConfig',
-        'getSiteIndexable',
```

Comment:
This was actually a different composable I introduced that was later moved to Site Config (https://nuxtseo.com/site-config/nitro-api/get-site-indexable).

Your fix is perfect though :+1: